practice-app/libs/getGeneralClasses.js

```diff
@@ -0,0 +1,81 @@
+var axios = require('axios');
+var querystring = require('querystring');
+var key = require('../credentials/key.json');
+var mysql      = require('mysql');
+var wikiData = require('./getDatafromWikiData');
+
+//gets label given a class_id
+function wiker(class_id){
+
+    var queryString = `SELECT  *  WHERE {  wd:${class_id} rdfs:label ?label .FILTER (langMatches( lang(?label), "en" ) ) } LIMIT 1`;
+    var url = 'https://query.wikidata.org/sparql';
+
+    return axios.post(url,
+        querystring.stringify({
+            query: queryString,
+            format: "json"
+        }), {
+            headers: {
+                "Content-Type": "application/x-www-form-urlencoded"
+            }
+     }); 
+}
+
+// inserts a item to the db
+async function insertClass(class_id,class_label) {
+    var connection = mysql.createConnection(key);
+
+    connection.connect(function(err){
+    if(!err) {
+        console.log("Database is connected ... nn");    
+    } else {
+        console.log(err);    
+    }
+    });
+
+    values = [];
+    values.push(class_id);
+    values.push(class_label);
+
+    var sql = 'INSERT IGNORE INTO Class_Table ( class_id, class_label) values (?,?)';
+    return new Promise((resolve, reject) => {
+        connection.query(sql, values, function (err, result) {
+            if (err) throw err;
+            resolve('item inserted');
+        }).on('error', function(e) {
+            reject(e.message);
+        });
+    }); 
+}
+
+//This function returns items from wikidata which has 16 or more subclasses
+async function getter(item_num){
+    var t=1;
```

Comment:
Please use more understandable variable name

Reply:
You are right about the varible and function names, I will change them.